core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20230829_mrb_add_sts_project.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="mtalbott" id="add_STS_GOOGLE_PROJECT_MRB_MIGRATION_STORAGE_TRANSFER_JOB">
+        <addColumn tableName="MULTIREGIONAL_BUCKET_MIGRATION_STORAGE_TRANSFER_SERVICE_JOB">
+            <column name="GOOGLE_PROJECT" type="VARCHAR(30)">
+                <constraints nullable="true"/>
```

Comment:
I made this nullable because I wasn't sure how to insert an environment-specific default. Sort of annoying that it has to be an `Option` as a result, but it seemed more straightforward than trying to work out how to get this to default to `broad-dsde-{ENV}`